_posts/2019-07-29-vim-basics.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: vim Basics
+date: 2019-07-29 22:28:02
+categories:
+author: bronzdoc
+---
+
+# VIM basics
+
+I would not bore you by telling you the history of vim, and the differences between vi and vim and all of its unix philosophy....
+If you are interested in that, you can give it a look [here](https://en.wikipedia.org/wiki/Vim_(text_editor).
+
+So all you need to know is that vim is a text editor... in my opinion one of vim strengths comes with how flexible it can be. It can be as simple or complex as you wish. It can serve you to edit a simple config file on a server or you can make it a complete code editing environment if you want.
```

Comment:
```suggestion
So all you need to know is that vim is a text editor. In my opinion one of vim's strengths comes with how flexible it can be. It can be as simple or complex as you wish. It can be used to edit a simple config file on a server or you can make it a complete code editing environment.
```